server/liveblog/blogs/request_membership.py

```diff
@@ -72,17 +71,19 @@ class MembershipResource(Resource):
         'default_sort': [('_updated', -1)]
     }
     resource_methods = ['POST']
-    privileges = {'POST': 'blogs'}
+    privileges = {'POST': 'request_membership'}
 
 
-class MembershipService(BlogService):
-    notification_key = 'request'
 
+class MembershipService(BaseService):
+    notification_key = 'request'
+  
     def on_create(self, docs):
         for doc in docs:
+            doc['original_creator'] = str(get_user().get('_id'))
             doc['message'] = "Please add me as a contributor to your blog"
```

Comment:
It isn't really needed. I think it can be deleted if @vladnicoara says it doesn't need it